### DISCUSSION TITLE
I'm using the Daily Long Time-lapses QR code but would like to attempt upload at certain time

### DISCUSSION THREAD

**Discussion by f3rn4ndrum5** [gopro/labs#310] [Q&A]
Hi and thanks in advance for your help!

We're shooting a construction site timelapse with three Hero10. ( https://gopro.github.io/labs/control/construction/ ) 

mPdP>06:59<18:00!645SQ~!07:00S!1R

There is AC power and wifi on site.

We start the timelapse at 07:00 and end at 18:00 and take 60 photos a day.

But this does not work wity auto-upload. The script crashes or goes in a wierd loop when it's on. 

The construction is 5 hours away and we leave the cameras shooting but there is no way to know if one fails (they have not), and we have to go every three weeks to download the data, clean the cameras and show it the QR code again.
 
Can an autoupload command be added to the script say at 19:00?

**Comment by dnewman-gpsw** [comment]
That type of script is no longer recommended as it is not reliable -- too many wake/shutdowns per day.

Try using a script like https://gopro.github.io/labs/control/dailytl/

e.g. !07:00NmTPp.300!S!39600E!U!1R  (this is 2X photo per day, but only one wake / shutdown per day.)


"The script crashes or goes in a wierd loop when it's on." Can you explain more on what you tried?

**Reply by f3rn4ndrum5** [reply]
When  https://gopro.github.io/labs/control/construction/  is running and auto-upload it's on, the camera turns off and on and on and off and does not take a picture.

 Is https://gopro.github.io/labs/control/dailytl/ video only? Can it take photos instead? 

Also what I like from https://gopro.github.io/labs/control/construction/ is that the timelapse starts right away if the code is read inside the timelapse window.

So in  https://gopro.github.io/labs/control/dailytl/ , the camera stays on all day? does this affect the life of the camera? this is a hot place and the camera is inside a pelican case that receives some shade but still.

thanks for your help

**Reply by dnewman-gpsw** [reply]
https://gopro.github.io/labs/control/construction/  Doesn't have an upload option, so how are you modifying it?

Yes, you can make is take Photo, the example I provided takes photos.

Timelapse modes are not high energy, this is the way to go.

**Reply by f3rn4ndrum5** [reply]
I don't modify https://gopro.github.io/labs/control/construction/ , just turn on auto upload in the camera settings and then it goes into the mentioned loop.

I'm going to try https://gopro.github.io/labs/control/dailytl/ for a couple of days and the different options, like taking video or photos, and also going to monitor heat from the camera.

Thanks for your help, means a lot.

f3
